src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -259,7 +259,7 @@ object GenomeReference {
   def apply(name: java.lang.String, contigs: java.util.ArrayList[String], lengths: java.util.HashMap[String, Int],
     xContigs: java.util.ArrayList[String], yContigs: java.util.ArrayList[String],
     mtContigs: java.util.ArrayList[String], parInput: java.util.ArrayList[String]): GenomeReference = {
-    val parRegex = """(\w+):(\d+)-(\d+)""".r
+    val parRegex = """(.+):(\d+)-(\d+)""".r
```

Comment:
I just checked the Locus/Interval parser which is this:

```def contig: Parser[String] = "\\w+".r```

Reply:
that should be fixed too, for sure.

Reply:
Reverting to `\w+` because the Interval parser doesn't work with `.+`. We can revisit this another time if needed.